kubernetes/README.md

```diff
@@ -60,8 +60,6 @@ configuration.api_key['authorization'] = 'YOUR_API_KEY'
 # Defining host is optional and default to http://localhost
 configuration.host = "http://localhost"
 
-# Defining host is optional and default to http://localhost
-configuration.host = "http://localhost"
```

Comment:
This file seems to be generated: https://github.com/kubernetes-client/python/commits/master/kubernetes/README.md. Perhaps this comes from the upstream openapi-generator?

Reply:
According to git blame, this part of README.md ( _lines 60 and 61_) was added by **OpenAPI generator v4.3.0**, but by mistake, _lines 63 and 64_ were added by **Yu Liao** when updating to version **1.26**.
although it was written 4 years ago by OpenAPI generator.

here is the screenshot : 

![image](https://github.com/kubernetes-client/python/assets/63799050/3cf9741b-0e1d-420a-b553-f9e73fe55eb8)